nightingale_perception/launch/movo_body_filter.launch

```diff
@@ -0,0 +1,24 @@
+<!-- launch file for point cloud laser scan conversion and filtering of robot body
+<launch>
+
+  <!-- Point cloud body filter node -->
+  <node name="pointcloud_filter" pkg="sensor_filters" type="pointcloud2_filter_chain">
+      <rosparam command="load" file="../config/movo_body_filter.yaml" />
```

Comment:
id recommend not doing relative paths like this. use absolute paths with the find keyword. see other launch files for examples